enclave-runtime/Cargo.toml

```diff
@@ -118,6 +118,7 @@ getrandom = { git = "https://github.com/integritee-network/getrandom-sgx", branc
 log = { git = "https://github.com/mesalock-linux/log-sgx" }
 env_logger = { git = "https://github.com/mesalock-linux/env_logger-sgx" }
 sp-io = { git = "https://github.com/integritee-network/sgx-runtime", branch = "master"}
+#sp-io = { path = "../../sgx-runtime/substrate-sgx/sp-io"}
```

Comment:
I think that can be removed now

Reply:
I prefer to leave it, as I wasted time getting it to work with the local code.

Reply:
I think it is better to have that in the patch section because this dependency will not overwrite the `sp-io`s in the enclave's dependencies. I am surprised that this even worked.

Just remember, as patches are only applied per workspace, you need to patch it in the enclave-runtime's toml. Patches in the root-workspace do not apply to the enclave.

Reply:
I don't understand. 
This is in the patch section for 
`[patch."https://github.com/paritytech/substrate"]`
and it is in the enclave-runtime's toml. 
sp-io is also in the patch for 
`[patch."https://github.com/integritee-network/sgx-runtime"]`, in the enclave-runtime's toml and in the root-workspace.
I think, all are required.

Reply:
Ah, sorry, I did not get that this is in the patch section. Sorry for that. Then it is ok, and I also think it is more convenient if they remain as comments, so we can simply uncomment them when needed.